grpc/src/main/proto/th2_grpc_lw_data_provider/queue_data_provider.proto

```diff
@@ -36,6 +36,10 @@ message MessageGroupsQueueSearchRequest {
   google.protobuf.Duration sync_interval = 4;
   string external_queue = 5;
   bool keep_alive = 6;
+  /**
+  Enables sending raw batches directly to external_queue. Sends raw directly first then sends raw to pins
+   */
+  bool send_raw_directly = 8;
```

Comment:
Maybe define additional property with array type for setting expected response: RAW, PARSED

Reply:
I don't we need an enumeration here